### PR TITLE
KOGITO-9405: Move from Kogito operator to SonataFlow Operator

### PR DESCRIPTION
**Pull request by ederign**
Binaries: https://drive.google.com/file/d/1ic5dRAwyyDTJ7PvPu7Q3W2gzV2xQQESi/view?usp=sharing

### CODE REVIEW

**Comment by ederign** [comment]
@caponetto as you saw on the channel, the dev images are broken for ARM. I've tested on @paulovmr 's machine and it worked.

**Review by paulovmr — APPROVED** [review]

**Review by caponetto — APPROVED** [review]

**Comment by ederign** [comment]
@zdrapela to test this tomorrow, you will have to install the new SonataFlow operator on your cluster: 

kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/operator.yaml

File: packages/kn-plugin-workflow/pkg/command/deploy.go [hunk 1]
```diff
@@ -91,13 +91,13 @@ func runDeployUndeploy(cmd *cobra.Command, args []string) error {
...
 
-	fmt.Printf("\n🎉 Kogito Serverless Workflow project successfully deployed.")
+	fmt.Printf("\n🎉 SonataFlow project successfully deployed.")
 
 	return nil
 }
 
 func deploy(cfg *DeployUndeployCmdConfig) error {
-	fmt.Printf("🛠 Deploying your Kogito Serverless project in namespace %s\n", cfg.NameSpace)
+	fmt.Printf("🛠  Deploying your SonataFlow project in namespace %s\n", cfg.NameSpace)
```
Comment (zdrapela) [inline packages/kn-plugin-workflow/pkg/command/deploy.go]:
```suggestion
	fmt.Printf("🛠 Deploying your SonataFlow project in namespace %s\n", cfg.NameSpace)
```

File: packages/kn-plugin-workflow/pkg/command/deploy_undeploy_common.go
Comment (zdrapela) [inline packages/kn-plugin-workflow/pkg/command/deploy_undeploy_common.go:1]: I believe that line 77 should be changed from this:
```
	fmt.Printf(" - ✅ Serverless workflow file found: %s\n", cfg.SWFFile)
```
to this:
```
	fmt.Printf(" - ✅ SonataFlow file found: %s\n", cfg.SWFFile)
```
And maybe, in the code, change the function `findServerlessWorkflowFile` to `findSonataFlowFile` and `cfg.SWFFile` to `cfg.SonataFlowFile`
And possibly on line 171 correct the error message to SonataFlow

File: packages/kn-plugin-workflow/pkg/command/run.go [hunk 2]
```diff
@@ -74,7 +74,7 @@ func run() error {
 		}
 		return nil
 	} else if common.IsQuarkusSWFProject() {
-		return fmt.Errorf("Looks like you are inside a Quarkus project. If that is the case, you should run it with \"quarkus run\" command.")
+		return fmt.Errorf("looks like you are inside a Quarkus project. If that is the case, you should run it with \"quarkus run\" command")
 	} else {
 		return fmt.Errorf("cannot find Serverless Workflow project")
```
Comment (zdrapela) [inline packages/kn-plugin-workflow/pkg/command/run.go]:
```suggestion
		return fmt.Errorf("cannot find SonataFlow project")
```

**Review by zdrapela — CHANGES_REQUESTED** [review]
Hi, I am not entirely sure about the change to SonataFlow, so please correct me if I am wrong.

Please see my inline comment on [deploy_undeploy_common.go](https://github.com/kiegroup/kie-tools/pull/1782/files#diff-b829c24be376de28e14c2c5da594c0c152ac52dac2e50f651585778b2d403cab).

I noticed that `README.md` and `index.js` mentions `Kogito Serverless Workflow`, maybe change that to `Sonata Flow`?

In `common/checks.go`, functions might be renamed to better align with the new name:
- `IsQuarkusSWFProject` -> `IsQuarkusSonataFlowProject`
- `IsSWFProject` -> `IsSonataFlowProject`

File: packages/kn-plugin-workflow/pkg/command/deploy.go [hunk 3]
```diff
@@ -91,13 +91,13 @@ func runDeployUndeploy(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("❌ ERROR: applying deploy: %w", err)
 	}
 
-	fmt.Printf("\n🎉 Kogito Serverless Workflow project successfully deployed.")
+	fmt.Printf("\n🎉 SonataFlow project successfully deployed.")
```
Comment (zdrapela) [inline packages/kn-plugin-workflow/pkg/command/deploy.go]:
```suggestion
	fmt.Printf("\n🎉 SonataFlow project successfully deployed.\n")
```

**Comment by ederign** [comment]
@zdrapela about error messages : "Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context."

Other issues are all fixed

**Review by zdrapela — APPROVED** [review]

**Pull request merged**